src/sentry/static/sentry/app/components/events/contextSummary.jsx

```diff
@@ -103,13 +104,13 @@ export class OsSummary extends React.Component {
   }
 }
 
-class UserSummary extends React.Component {
+export class UserSummary extends React.Component {
   static propTypes = {
     data: PropTypes.object.isRequired,
   };
 
   render() {
-    const user = this.props.data;
+    const user = removeFilterMaskedEntries(this.props.data);
```

Comment:
Why do this for the user summary and context specifically? Note that relay will emit entirely different, non-constant mask values so this will need updating.

For context summaries at the top this makes sense to me but I don't see why we'd do this for contexts displays at the bottom of the event.

Reply:
It seems to me that the user *summary* will not show filtered values though, so it's not just for the avatar, right?

Reply:
Okay, fair enough - you're right. I was thinking of the user summary just being the avatar, but you're reminding me there's text there, too. (I originally worked on this long enough ago that I'd forgotten!)

Because only some parts of the data show in the user summary (depending on what's available), I wanted to make sure that if we had good data, it was what showed. Since the user summary is just that - a summary - it's never complete user data anyway, so it seemed okay to get rid of the filtered stuff, since everything, including filtered fields, is available down below in the User context section.

Reply:
That makes sense.